tests/integration/operator/main_test.go

```diff
@@ -0,0 +1,30 @@
+//  Copyright Istio Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package operator
```

Comment:
The prow targets are running folders under test/integration/X explicitly. So to run this you would either need to make a new job (probably not desired) or move it into an existing folder. The layout of the folders isn't great right now unfortunately

Reply:
meaning this is not actually running in the PR, by the way

Reply:
yep I understand that this is not triggered right now, why you said adding a new job is not desired, I actually cannot find a suitable existing package I can add to, I would prefer adding new package and new prow job?

Reply:
Yeah, our package layout is not ideal. The problem is each test suite has a ~8 minute overhead, so adding a whole new suite for 1 tests, which takes like 2 minutes, means we are wasting 80% of the time, so its better to avoid it. If we start adding more tests it may make more sense?

Reply:
I see, I think the tests duration would be increased after I add more IOP crs to test

Reply:
+1 for adding a new suite. the goal is to have many more tests